pkg/backend/cancellation_scope.go

```diff
@@ -49,15 +50,15 @@ func (cancellationScopeSource) NewScope(events chan<- engine.Event, isPreview bo
 
 	c := &cancellationScope{
 		context: cancelContext,
-		sigint:  make(chan os.Signal),
+		sigint:  make(chan os.Signal, 1),
```

Comment:
The channel for `signal.Notify` should be buffered https://pkg.go.dev/os/signal#Notify

Reply:
Worth adding this as a comment to the code? Seems like a useful thing to call out.

Reply:
Yeah, good idea